include/juice/juice.h

```diff
@@ -64,6 +64,16 @@ typedef void (*juice_cb_gathering_done_t)(juice_agent_t *agent, void *user_ptr);
 typedef void (*juice_cb_recv_t)(juice_agent_t *agent, const char *data, size_t size,
                                 void *user_ptr);
 
+typedef struct juice_mux_incoming {
+	const char *ufrag;
+	const char *pwd;
```

Comment:
You can't retrieve the password here, it's not in the STUN probe (otherwise anybody could read it).